wrappers/core18.go

```diff
@@ -64,8 +64,8 @@ func writeSnapdToolingMountUnit(sysd systemd.Systemd, prefix string, opts *AddSn
 
 	// TODO: the following comment is wrong, we don't need RequiredBy=snapd here?
 
-	// Not using AddMountUnitFile() because we need
-	// "RequiredBy=snapd.service"
+	// Not using EnsureMountUnitFile() because we need
+	// "WantedBy=snapd.service"
```

Comment:
I think this change should go to a separate commit, with some explanation.

Reply:
That should not be here. I do not even understand the comment.

Reply:
I have removed this change.